LocalDynamoDb.Tests/LocalDynamoDb.Tests.csproj

```diff
@@ -0,0 +1,30 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <TargetFramework>netcoreapp2.1</TargetFramework>
+        <IsPackable>false</IsPackable>
+    </PropertyGroup>
+
+    <ItemGroup>
+        <PackageReference Include="AWSSDK.Core" Version="3.3.27" />
+        <PackageReference Include="AWSSDK.DynamoDBv2" Version="3.3.14.5" />
+        <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.8.0" />
+        <PackageReference Include="xunit" Version="2.3.1" />
+        <PackageReference Include="xunit.runner.visualstudio" Version="2.3.1" />
+    </ItemGroup>
+
+    <ItemGroup>
+        <Content Include="dynamodblocal\**">
```

Comment:
`<Content Include="dynamodblocal\**" CopyToOutputDirectory="Always" />` would be neater.